e2e/cli-e2e/tests/collect.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('CLI collect', () => {
   it('should run ESLint plugin and create report.json', async () => {
     const { code, stderr } = await executeProcess({
       command: 'npx',
-      args: ['../../dist/packages/cli', 'collect'],
+      args: ['../../dist/packages/cli', 'collect', '--no-progress'],
```

Comment:
It's cool that the negated syntax works :slightly_smiling_face:

Reply:
We picked the right lib at the beginning it seems. I was also impressed that the "dot-syntax" just works